spec/features/force_user_id_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe "Acts as a different User in request" do
+  let(:user) { FactoryGirl.create(:user, client_slug: 'gsa18f') }
+
+  it "respects session for current_user" do
+    wo = FactoryGirl.create(:ncr_work_order, :with_approvers)
+    login_as(wo.proposal.requester)
+    visit '/me'
+    expect(page.find('h2')).to have_content(wo.proposal.requester.email_address)
+  end
+
+  # can't use with_env here because we need to use factory object as env var value
```

Comment:
```
diff --git a/spec/features/force_user_id_spec.rb b/spec/features/force_user_id_spec.rb
index 993e2d0..fc49537 100644
--- a/spec/features/force_user_id_spec.rb
+++ b/spec/features/force_user_id_spec.rb
@@ -8,13 +8,13 @@ describe "Acts as a different User in request" do
     expect(page.find('h2')).to have_content(wo.proposal.requester.email_address)
   end
 
-  # can't use with_env here because we need to use factory object as env var value
-  it "respects FORCE_USER_ID to override current_user" do
-    ENV['FORCE_USER_ID'] = user.id.to_s
-    wo = FactoryGirl.create(:ncr_work_order, :with_approvers)
-    login_as(wo.proposal.requester)
-    visit '/me'
-    expect(page.find('h2')).to have_content(user.email_address)
+  with_env('FORCE_USER_ID', user.id.to_s) do
+    it "respects FORCE_USER_ID to override current_user" do
+      wo = FactoryGirl.create(:ncr_work_order, :with_approvers)
+      login_as(wo.proposal.requester)
+      visit '/me'
+      expect(page.find('h2')).to have_content(user.email_address)
+    end
   end
 
 end
```

yields this error:

```
in `method_missing': `user` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). (RSpec::Core::ExampleGroup::WrongScopeError)
```